src/main/java/seedu/address/logic/commands/session/EditSessionCommand.java

```diff
@@ -73,7 +75,15 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_DUPLICATE_SESSION);
         }
 
+        for (int i = 0; i < lastShownList.size(); i++) {
```

Comment:
should this check with all sessions in fitego, instead of just those that are shown?

Reply:
Okay i'll change that, makes sense